File: Serene/Serene.Script/Meeting/MeetingAgendaType/MeetingAgendaTypeDialog.cs

namespace Serene.Meeting
{
    using jQueryApi;
    using Serenity;
    using System.Collections.Generic;

    [IdProperty("MeetingAgendaTypeId"), NameProperty("Name"), IsActiveProperty("IsActive")]
    [FormKey("Meeting.MeetingAgendaType"), LocalTextPrefix("Meeting.MeetingAgendaType"), Service("Meeting/MeetingAgendaType")]
    public class MeetingAgendaTypeDialog : EntityDialog<MeetingAgendaTypeRow>
    {
    }
}

File: Serene/Serene.Core/Modules/Meeting/MeetingAgendaType/MeetingAgendaTypeDialog.ts

namespace Serene.Meeting {

    @Serenity.Decorators.registerClass()
    @Serenity.Decorators.responsive()
    export class MeetingAgendaTypeDialog extends Serenity.EntityDialog<MeetingAgendaTypeRow, any> {
        protected getFormKey() { return MeetingAgendaTypeForm.formKey; }
        protected getIdProperty() { return MeetingAgendaTypeRow.idProperty; }
        protected getLocalTextPrefix() { return MeetingAgendaTypeRow.localTextPrefix; }
        protected getNameProperty() { return MeetingAgendaTypeRow.nameProperty; }
        protected getService() { return MeetingAgendaTypeService.baseUrl; }

        protected form = new MeetingAgendaTypeForm(this.idPrefix);

    }
}